roottest/root/io/constmembers/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST()
+if(broken_tests)
```

Comment:
With this type of migration from `Makefile` to `CMake` we lose the existing warning/reminder (which to be fair was being overlooked).  We should probably open a generic issue point to the existence of this tests that will be since only with `-Dbroken_tests=ON` (ideally we would have one issue per failing test/directory).   

My apologies if those issues already exists.

Reply:
The windows one is more systemic choice.  The known to fail every where test are more focused and pre-dated the github issue system.  At the time it was thought that 'recording' their failure by a warning was sufficient, nowadays, I believe it is not and that we need to record it (either coarse grain by having an issue mentioning `broken_tests` or pin-pointed by having an issue by test.  The advantage is that contributor will then have some visibility on the fact that those outstanding issue exist and can be picked up to work on.